releasenotes/notes/update-lib-injection-to-use-safe-instrumentation-patching-e2035b51fafb793b.yaml

```
@@ -0,0 +1,6 @@
+---
+other:
+  - |
+    single-step instrumentation: Updates library injection guardrails to use safe instrumentation patching feature ``DD_TRACE_SAFE_INSTRUMENTATION_ENABLED``. 
+    This change ensures that instrumentation patching is only applied to for supported versions of packages, leaving unsupported
+    package versions unpatched.
```

nit: formatting? I'm not sure if there's a character limit per line.
```suggestion
    This change ensures that instrumentation patching is only applied to for supported versions of packages, leaving unsupported package versions unpatched.
```